src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -532,6 +585,9 @@ impl<'a, 'tcx> DocFolder for LinkCollector<'a, 'tcx> {
                 } else if link.starts_with("macro@") {
                     kind = Some(MacroNS);
                     link.trim_start_matches("macro@")
+                } else if link.starts_with("derive@") {
```

Comment:
I'm wondering: is there any reason behind supporting `derive@` too? (Not against it, just curious)